galaxy_api/api/ui/viewsets/namespace.py

```diff
@@ -3,11 +3,19 @@
 
 from rest_framework import mixins
 from rest_framework import viewsets
+from rest_framework import status
+
 from rest_framework.settings import api_settings
+from rest_framework.response import Response
 
 from galaxy_api.api import models, permissions
 from galaxy_api.api.ui import serializers
 
+from galaxy_api.auth import models as auth_models
+
+
+RH_ACCOUNT_SCOPE = 'rh-identity-account'
```

Comment:
Minor: This constant must be already defined in authentication modules. If yes, please reuse it.